samples/create_cluster.js

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START gke_create_cluster]
```

Comment:
Is this sample going to be displayed anywhere? Usually region tags help embed samples in `cloud.google.com`, but I'm not sure if this is the correct segment of sample you want displayed. Usually, it just wraps the main function.

Reply:
So ideally we want to surface it in the sample browser. Hence, the region tag encompassing as much of it as possible

Reply:
Fixed in https://github.com/googleapis/nodejs-cloud-container/pull/519/commits/ee670d950b4f5a8e53ec494e4448b2bd2ca38e6b.

Moved the region tags to encompass everything inside the `main`